homework-submission/my-project/src/AddForm.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import DatePicker from 'react-datepicker';
+import 'react-datepicker/dist/react-datepicker.css';
+
+
+class AddTodo extends React.Component {
+    state = {
+        description: '',
+        deadline: '',
+        done: false,
+
+    }
+    newTodo = (e) => {
+        this.setState({
+            [e.target.id]: e.target.value
```

Comment:
If you are not using `id`'s in your Css then no need for them in your React application, you may use `name` attribute ?